### DISCUSSION TITLE
I am not allowed to see my company repository

### DISCUSSION THREAD

**Discussion by ThaerAlkassab** [community/community#124629] [Repositories]
The GitHub keeps giving me this message
We are unable to provide this service.
It appears your account may be based in a U.S.-sanctioned region. As a result, we are unable to provide private repository services and paid services for your account. GitHub has preserved, however, your access to certain free services for public repositories. If your account has been flagged in error, and you are not located in or resident in a sanctioned region, please file an appeal. Please read about GitHub and Trade Controls for more information.
I used to live in Syria, but now I am a resident of Hungary.
Please solve this problem for me as soon as possible.

**Comment by shivamgaur99** [comment]


**Reply by queenofcorgis** [reply]
@shivamgaur99 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
